packages/drpc/src/repository/DrpcMessageRecord.ts

```diff
@@ -0,0 +1,75 @@
+import type { DrpcRole } from '../DrpcRole'
+import type { DrpcState } from '../DrpcState'
+import type { DrpcRequest, DrpcResponse } from '../messages'
+import type { RecordTags, TagsBase } from '@credo-ts/core'
+
+import { BaseRecord, CredoError, utils } from '@credo-ts/core'
+
+export type CustomDrpcMessageTags = TagsBase
+export type DefaultDrpcMessageTags = {
+  connectionId: string
+  threadId: string
+}
+
+export type DrpcMessageTags = RecordTags<DrpcMessageRecord>
+
+export interface DrpcMessageStorageProps {
+  id?: string
+  connectionId: string
+  role: DrpcRole
+  tags?: CustomDrpcMessageTags
+  message: DrpcRequest | DrpcResponse
```

Comment:
IMHO it would be clearer if we'd store separate `request?: DrpcRequest` and `response?: DrpcResponse`

The term `message` is mostly used (in credo context) for DIDComm messages, so using that here for another kind of object can be confusing.